.github/workflows/ci.yml

```diff
@@ -47,3 +47,76 @@ jobs:
           architecture: "x64"
       - run: pip install flake8 flake8-print
       - run: flake8 --enable=T .
+
```

Comment:
Should we get a coverage report?

Reply:
I don't see the point yet, since we don't have a goal for coverage

Reply:
We would see the places which we are not covering and probably are hiding possible errors/bugs.

Reply:
We can try to add this as a part of #9 epic, not in this PR.

Reply:
Okay, then please issue a PR related to coverage report to see how it goes.